tests/src/whisk/core/cli/test/WskCoreBasicTests.scala

```diff
@@ -0,0 +1,210 @@
+                (action, _) => action.create(sampleAction, file, kind = Some("nodejs"), shared = Some(true))
+            }
+            val fullyQualifiedActionName = s"/$otherNamespace/notallowed"
+            wsk.action.create(fullyQualifiedActionName, None,
+                              update = true, shared = Some(true), expectedExitCode = FORBIDDEN)(originWskProps)
+    }
+
+    behavior of "Wsk Package CLI"
+
+    it should "list shared packages" in withAssetCleaner(otherWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, _) =>
+                    pkg.create(samplePackage, parameters = Map("a" -> "A".toJson), shared = Some(true),
+                               update = true)
+            }
+            val fullyQualifiedPackageName = s"/$otherNamespace/$samplePackage"
+            val result = wsk.pkg.list(Some(s"/$otherNamespace"))(originWskProps).stdout
+            result should include regex (fullyQualifiedPackageName + """\s+shared""")
+    }
+
+    it should "list shared package actions" in withAssetCleaner(otherWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, _) =>
+                    pkg.create(samplePackage, parameters = Map("a" -> "A".toJson), shared = Some(true),
+                               update = true)
+            }
+            assetHelper.withCleaner(wsk.action, sampleAction) {
+                val file = Some(TestUtils.getTestActionFilename("empty.js"))
+                (action, _) => action.create(sampleAction, file, kind = Some("nodejs"), shared = Some(true))
+            }
+            val fullyQualifiedPackageName = s"/$otherNamespace/$samplePackage"
+            val fullyQualifiedActionName = s"/$otherNamespace/$sampleAction"
+            val result = wsk.action.list(Some(fullyQualifiedPackageName))(originWskProps).stdout
+            result should include regex (fullyQualifiedActionName + """\s+shared""")
+    }
+
+    it should "create a package binding" in withAssetCleaner(otherWskProps) {
+        (wp, assetHelper) =>
+            assetHelper.withCleaner(wsk.pkg, samplePackage) {
+                (pkg, _) =>
+                    pkg.create(samplePackage, parameters = Map("a" -> "A".toJson), shared = Some(true),
+                               update = true)
+            }
+            val fullyQualifiedPackageName = s"/$otherNamespace/$samplePackage"
+            val name = "bindPackage"
+            val provider = fullyQualifiedPackageName
+            val annotations = Map("a" -> "A".toJson, WhiskPackage.bindingFieldName -> "xxx".toJson)
+            assetHelper.withCleaner(wsk.pkg, name) {
+                (pkg, _) =>
+                    pkg.bind(provider, name, annotations = annotations)
+            }
+            val stdout = wsk.pkg.get(name).stdout
+            stdout should include regex (""""key": "a"""")
```

Comment:
use `wsk.parse as json` and then you can tighten this test to check parameters vs annotations separately (which you should do to confirm parameter merging is ok and that annotation are properly reported from the binding).